tests/cases/fourslash/formattingOptionsChangeJsx.ts

```diff
@@ -1,17 +1,22 @@
 ///<reference path="fourslash.ts"/>
 
 //@Filename: file.tsx
-/////*InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces*/<Madoka homu={                true    } saya={   (true)  } />;
+/////*InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces1*/<Madoka homu={                true    } saya={   (true)  } />;
+////
+////<PrimaryButton
+////    /*InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces2*/{    ...this._getButtonProps()    }
+/////>
 
-runTest("InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces", "<Madoka homu={ true } saya={ (true) } />;", "<Madoka homu={true} saya={(true)} />;");
+runTest("InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces1", "<Madoka homu={ true } saya={ (true) } />;", "<Madoka homu={true} saya={(true)} />;");
+runTest("InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces2", "    { ...this._getButtonProps() }", "    {...this._getButtonProps()}");
 
 
 function runTest(propertyName: string, expectedStringWhenTrue: string, expectedStringWhenFalse: string) {
     // Go to the correct file
     goTo.marker(propertyName);
 
     // Set the option to false first
-    format.setOption(propertyName, false);
+    format.setOption(propertyName.slice(0, -1), false);

```

Comment:
Instead of giving the markers this silly name, just use `const propertyName = "InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces";` and use `markerName` as a parameter.